source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACFramework.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ADCIOSFramework.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/2/17.
```

Comment:
>Created by jwoo on 6/2/17. [](start = 4, length = 26)

We don't normally put file history in the file. Source control keeps track of that.